### ISSUE TITLE
Incorrect type on errors field in Braintree ValidatedResponse

### ISSUE THREAD

**Issue by RowanG1Bilue** [braintree/braintree_node#213]
For the interface of Braintree below:

```
 interface ValidatedResponse<T> {
        success: boolean;
        errors: ValidationErrorsCollection;
...
}
```

When we get a success is true response, the errors field is undefined, whereas in the type declaration, the errors field has to be defined. This can potentially cause unexpected errors to be thrown, if a developer happens to access the errors on the success path (which generally should not happen in principle).

### General information

- Braintree v3.5.0 
- Sandbox
- Node v17.3.0

**Comment by hollabaq86** [comment]
👋 @RowanG1Bilue thanks for reaching out. This SDK [doesn't have type support](https://github.com/braintree/braintree_node/issues/176) yet 🙃 I think the root of your issue is in a different library that uses Braintree Node?

**Comment by crookedneighbor** [comment]
More than likely, the problem is in the Definitely Typed repo: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/braintree

Closed as a duplicate of #176
